templates/.circleci/config.yml.tpl

```diff
@@ -111,10 +120,43 @@ workflows:
     when: << pipeline.parameters.rebuild_cache >>
     jobs:
       - shared/save_cache: *test
+  {{- if stencil.Arg "releaseOptions.autoPrereleases" }}
+
+  auto-release-rc:
+    triggers:
+      - schedule:
+        {{- $cronTime := "0 19 * * 2"}}
+        {{- if not ( empty (stencil.Arg "releaseOptions.prereleasesCron" )) }}
+        {{- $cronTime = stencil.Arg "releaseOptions.prereleasesCron" }}
+        {{- end }}
+          # Set the cron time in service manifest with releaseOptions.prereleasesCron
+          # The deault time is 12pm every Tuesday(PST)
```

Comment:
issue:
* this comment is only accurate if `releaseOptions.prereleasesCron` isn't set to a custom value.
* When using the default value, this comment is only accurate during DST. Outside of DST it would be 11am PT.

I'd just get rid of the comment.

Reply:
Make sense, I will get rid of it.